### PR TITLE
Add isTrialOnly readonly property

### PR DESCRIPTION
**Pull request by lexerom**
Adds isTrialOnly for GET plan or list of plans

### CODE REVIEW

File: openapi/components/schemas/Common/CommonPlan.yaml [hunk 1]
```diff
@@ -61,6 +61,10 @@ properties:
         format: double
       period:
         $ref: ../Plans/PlanPeriod.yaml
+  isTrialOnly:
+    type: boolean
+    description: If a plan has no recurring settings, but has a trial.
```
Comment (VladimirBerdnik) [inline openapi/components/schemas/Common/CommonPlan.yaml:66]:
```suggestion
    description: Whether a plan has a trial without recurring instructions.
```

**Comment by creocoder** [comment]
Why we need that new property? Plan can be created without recurring instructions from start.

**Review by creocoder — APPROVED** [review]

**Comment by lexerom** [comment]
devs please review

**Review by removed-account-123 — APPROVED** [review]
👍

**Pull request merged**